packages/prisma/prisma/schema.prisma

```diff
@@ -2,33 +2,38 @@ datasource db {
   provider = "sqlite"
   url      = "file:./dev.db"
 }
+
 generator client {
   provider = "prisma-client-js"
 }
+
 model Account {
-  id                 String    @id @default(cuid())
+  id                 String  @id @default(cuid())
   userId             String
-  providerType       String
-  providerId         String
-  providerAccountId  String
-  refreshToken       String?
-  accessToken        String?
-  accessTokenExpires DateTime?
-  createdAt          DateTime  @default(now())
-  updatedAt          DateTime  @updatedAt
-  user               User      @relation(fields: [userId], references: [id])
-  @@unique([providerId, providerAccountId])
+  type               String
+  provider           String
+  refresh_token      String?
```

Comment:
Nitpick: is there a reason why some fields are in snake_case now? Prisma's docs [use camelCase for their examples](https://www.prisma.io/docs/about/about-the-docs#naming-conventions-for-tables-and-columns), and this seems to be true of most Prisma codebases I've seen.

Reply:
Mostly because the OAuth spec defines these as snake_case, and I thought it would make things easier if we just forward them "as-is".